### PR TITLE
Use OLM 0.17.0 instead of 0.14.1 in the OLM addon.

### PR DESCRIPTION
**Pull request by AndrienkoAleksandr**
Signed-off-by: Oleksandr Andriienko <oandriie@redhat.com>

### CODE REVIEW

File: pkg/minikube/assets/addons.go [hunk 1]
```diff
@@ -290,11 +290,13 @@ var Addons = map[string]*Addon{
 			"olm.yaml",
 			"0640"),
 	}, false, "olm", map[string]string{
-		"OLM":                        "operator-framework/olm:0.14.1@sha256:0d15ffb5d10a176ef6e831d7865f98d51255ea5b0d16403618c94a004d049373",
-		"UpstreamCommunityOperators": "operator-framework/upstream-community-operators:07bbc13@sha256:cc7b3fdaa1ccdea5866fcd171669dc0ed88d3477779d8ed32e3712c827e38cc0",
+		"OLM":                        "operator-framework/olm@sha256:de396b540b82219812061d0d753440d5655250c621c753ed1dc67d6154741607",
+		"UpstreamCommunityOperators": "operatorhubio/catalog:latest",
```
Comment (medyagh) [inline pkg/minikube/assets/addons.go:294]: please avoid using :latest all the images must have a sha

File: pkg/minikube/assets/addons.go [hunk 2]
```diff
@@ -290,11 +290,13 @@ var Addons = map[string]*Addon{
...
+		"ConfigMapServer":            "operator-framework/configmap-operator-registry:latest",
```
Comment (medyagh) [inline pkg/minikube/assets/addons.go:295]: same here

File: deploy/addons/olm/olm.yaml.tmpl [hunk 3]
```diff
@@ -78,9 +78,9 @@ spec:
           command:
           - /bin/olm
           args:
-          - -namespace
+          - --namespace
```
Comment (medyagh) [inline deploy/addons/olm/olm.yaml.tmpl:81]: have u verified that the extra - is needed ? was not working before without it ?

**Review by medyagh — CHANGES_REQUESTED** [review]

**Comment by medyagh** [comment]
@AndrienkoAleksandr are you still working on this ?

**Comment by AndrienkoAleksandr** [comment]
> @AndrienkoAleksandr are you still working on this ?

Hello, sorry I will continue on nearest Friday 12th.

File: deploy/addons/olm/olm.yaml.tmpl (hunk 3 above)
Comment (AndrienkoAleksandr) [reply deploy/addons/olm/olm.yaml.tmpl:81]: I think for OLM 0.17.0 should be doube `--`
olm --version
OLM version: 0.17.0
git commit: 5eb1fa6ad6ecfbf1263a145e54e5bd181a078097

olm --help
Usage of olm:
      --debug                                     use debug log level
      --feature-gates mapStringBool               A set of key=value pairs that describe feature gates for alpha/experimental features. Options are:
                                                  AllAlpha=true|false (ALPHA - default=false)
                                                  AllBeta=true|false (BETA - default=false)
      --interval duration                         wake up interval (default 5m0s)
      --namespace string                          namespace where cleanup runs
      --profiling                                 serve profiling data (on port 8080)
      --tls-cert string                           Path to use for certificate key (requires tls-key)
      --tls-key string                            Path to use for private key (requires tls-cert)
      --version                                   displays olm version
      --watchedNamespaces -watchedNamespaces=""   comma separated list of namespaces for olm operator to watch. If not set, or set to the empty string (e.g. -watchedNamespaces=""), olm operator will watch all namespaces in the cluster.
      --writePackageServerStatusName string       ClusterOperator name in which to write status for package API server, set to "" to disable.
      --writeStatusName string                    ClusterOperator name in which to write status, set to "" to disable.

**Comment by AndrienkoAleksandr** [comment]
I will reopen this pr later. I have to check some things.

**Pull request closed without merging**